src/plugins/content_management/common/rpc/bulk_get.ts

```diff
@@ -27,9 +28,13 @@ export const bulkGetSchemas: ProcedureSchemas = {
   ]),
 };
 
-export interface BulkGetIn<T extends string = string, Options extends object = object> {
+export interface BulkGetIn<T extends string = string, Options extends void | object = object> {
   contentTypeId: T;
   ids: string[];
   version?: Version;
   options?: Options;
 }
+
+export interface BulkGetResult<T = unknown, M = void> {
+  hits: Array<GetResult<T, M>>;
```

Comment:
should this be `items`?

Reply:
Talked with @vadimkibana and he wanted to have it similar to `search` so that's what we came with.